packages/core/src/platforms/Platform.ts

```
@@ -291,6 +295,21 @@ export abstract class Platform {
     return path.join('.');
   }
 
+  /* istanbul ignore next */
```

i dont the ignore comments much in here, but note we test similar things in here

https://github.com/mikro-orm/mikro-orm/blob/master/tests/DatabaseDriver.test.ts

it might also be better to throw instead of provide default that is not really a good default :] (at least it looks like you override this method in each supported driver)

---

Ah I see. Good suggestion.